From 97921274dedc570a5d349e2b6399d3bfccc6eb91 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rog=C3=A9rio=20Brito?= <rbrito@ime.usp.br>
Date: Mon, 17 Dec 2018 18:17:17 -0200
Subject: [PATCH 11/27] Revert "Skip compression of bicolor images with
 non-jbig2 compressors."

Let's try to see how pts's new method works.

This reverts commit 31eb7e4dfbc29d445ed43f1f7d1114a7325093ca.
---
 lib/pdfsizeopt/main.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/lib/pdfsizeopt/main.py b/lib/pdfsizeopt/main.py
index 6b0c1dc..2345178 100644
--- a/lib/pdfsizeopt/main.py
+++ b/lib/pdfsizeopt/main.py
@@ -7745,12 +7745,6 @@ class PdfData(object):
             cmd_name = GetCmdName(cmd_pattern)
             if not cmd_name or cmd_name in ('sam2p_pr', 'sam2p_np'):
               continue
-            # We skip compressing black-and-white images with compressors
-            # other than jbig2, since they usually (always in my experience)
-            # lose in compression and they take *waaaaay* too much time,
-            # especially if we have color images that we want optimized highly.
-            if ('jbig2' not in cmd_name) and (obj_images[-1][1].bpc == 1):
-              continue
             if cmd_name in cmd_names_used:
               i = 2
               while 1:
-- 
2.30.1

